### PR TITLE
ryujinx: fix .desktop

### PR DESCRIPTION
**Pull request by davegallant**
## Description of changes

Update the .desktop file so that it launches the application.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by wegank** [comment]
@ofborg build ryujinx ryujinx.passthru.tests

File: pkgs/by-name/ry/ryujinx/package.nix [hunk 1]
```diff
@@ -98,7 +98,7 @@ buildDotnetModule rec {
     install -D ../misc/Logo.svg $out/share/icons/hicolor/scalable/apps/Ryujinx.svg
 
     substituteInPlace $out/share/applications/Ryujinx.desktop \
-      --replace "Ryujinx %f" "$out/bin/Ryujinx %f"
+      --replace "Ryujinx.sh %f" "$out/bin/Ryujinx.sh %f"
```
Comment (wegank) [inline pkgs/by-name/ry/ryujinx/package.nix:101]: Wait, does `$out/bin/Ryujinx.sh` exist?

Comment (davegallant) [reply pkgs/by-name/ry/ryujinx/package.nix:101]: Yes. 
https://github.com/Ryujinx/Ryujinx/blob/3924bd1a4364455ab8a5747e3cb0b3000dbaa589/distribution/linux/Ryujinx.desktop#L6

Comment (wegank) [reply pkgs/by-name/ry/ryujinx/package.nix:101]: I mean, after building the package, what are the files in the `result/bin` folder? Did you see a `Ryujinx.sh`?

Comment (artemist) [reply pkgs/by-name/ry/ryujinx/package.nix:101]: Ryujinx.sh is a wrapper script. We might want to add it (it sets an environment variable, see [the source](https://github.com/Ryujinx/Ryujinx/blob/master/distribution/linux/Ryujinx.sh)) but don't currently

Comment (davegallant) [reply pkgs/by-name/ry/ryujinx/package.nix:102]: Ah, missed it. Added!
```
❯ ls /nix/store/kaxzr63y8dk52pwhqfxfvpnsk4rbf70d-ryujinx-1.1.1217/bin
ryujinx  Ryujinx  Ryujinx.Headless.SDL2  Ryujinx.sh
```

**Review by wegank — APPROVED** [review]

**Pull request merged**